### PR TITLE
CDAP-5765 fix es batch sink logging dependency

### PR DESCRIPTION
**Pull request by albertshau**
Added a dependency to jcl-over-slf4j to fix class not found
errors on logging classes.

### CODE REVIEW

File: elasticsearch-plugins/pom.xml [hunk 1]
```diff
@@ -85,6 +85,11 @@
       <groupId>commons-httpclient</groupId>
       <artifactId>commons-httpclient</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jcl-over-slf4j</artifactId>
+      <version>1.7.12</version>
```
Comment (gokulavasan) [inline elasticsearch-plugins/pom.xml]: Wondering if we should consolidate version numbers in the main pom?

Comment (albertshau) [inline elasticsearch-plugins/pom.xml]: ok can do.

**Comment by albertshau** [comment]
@gokulavasan fixed, can you take another look?

**Comment by gokulavasan** [comment]
LGTM 👍

**Pull request merged**